### DISCUSSION TITLE
Android SDK publishes own stream when you try to play remote stream - Publish/Play

### DISCUSSION THREAD

**Discussion by FarhodKurbonov** [ant-media/Ant-Media-Server#6348] [General]
Issue: When user tries to play already published stream by streamID, it plays remote streams audio but publishes local stream. As a result user sees own image but listens to remote stream audio

Use Case
1) User A: Launch webrtc-android-sample-app and make stream to publish
2) User B: Launch webrtc-android-sample-app and try to play already published stream of User A
Result: User B can see own local stream from camera but listens to remote stream audio. 

Tested Ant media Enterprise Server versions 2.8.0, 2.8.2, 2.9.0 
Tested Android WebRTC Framework - 2.8.3-SNAPSHOT 

We tested it on https://meet.antmedia.io/WebRTCAppEE/publish/play.html everything works as expected but when you use Android SDK or launch sample app user bumps into issue as described above

**Comment by Mohit-3196** [comment]
Hi @FarhodKurbonov,
Thank you for reporting.
I will check it and get back to you.

Thank you.

**Reply by mekya** [reply]
Hi @Mohit-3196,

Could you please update what the latest status is?

Cheers
Oz

**Reply by Mohit-3196** [reply]
Hello @mekya abi and @FarhodKurbonov,
Sorry for missing this. It's my bad.

I will test it this week and update it.

Thank you,
Mohit

**Reply by Mohit-3196** [reply]
Hi @FarhodKurbonov,
How are you?

I just tested this scenario with the sample publish and play and I did not face any such issue.
I tried both on a mobile device as well as the emulator and I was only playing the remote stream.

I tested it on the latest AMS version 2.9.0 and the 2.9.1-SNAPSHOT version of the SDK.
![Screenshot 2024-06-28 213958](https://github.com/ant-media/Ant-Media-Server/assets/86982446/9859bc3c-e275-4979-b76e-f007ff54bebf)

Can you please check this again on your side?
If you see any issues, we can connect and check as well.

Cheers,
Mohit
